department-of-veterans-affairs/vets-website
# VACMS 18319 - update API call

**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- Call was being made to v0 lighthouse via /v1/facilities/va endpoint. Switched to /facilities_api/v2/va
- Behind the flipper: use_facilities_api_v2_for_cerner_cta
- Sitewide Facilities

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18319

## Testing done

- Manual testing
- Check the CernerCallToAction Widget wherever it appears, see that it is making a POST request to /facilities_api/v2/va now

## Screenshots
N/A

## What areas of the site does it impact?

src/applications/static-pages/health-care-manage-benefits/get-medical-records-page/components/AuthContent/index.js
src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/AuthContent/index.js
src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/AuthContent/index.js
src/applications/static-pages/health-care-manage-benefits/view-test-and-lab-results-page/components/AuthContent/index.js

## Acceptance criteria

-[x] Uses the v2 endpoint

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable N/A).
- [ x No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Screenshot of the developed feature is added(N/A)
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed (N/A)

### Error Handling


### Authentication


## Requested Feedback

Check that the CTA still functions in logged in scenarios on the pages listed above
